.github/workflows/rust.yml

```diff
@@ -13,23 +13,19 @@ jobs:
       - name: Checkout branch
         uses: actions/checkout@v2
 
-      - name: Set variables
-        run: |
-          echo "TOOLCHAIN=$(rustup show active-toolchain | cut -d " " -f1)" >> $GITHUB_ENV
-
       - name: Install toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ env.TOOLCHAIN }}
```

Comment:
I couldn't find any usages of `TOOLCHAIN` outside this script. I was a bit concerned it could be used else such as the `DockerFile` or our deployment script. It seems that it's only used here, but if you can double check in case I missed something, that'd be great!

Reply:
It's not being used outside of this file but it is also set for the `tests` job further below.